tool/docker/fedora/Dockerfile

```diff
@@ -37,15 +37,14 @@ ENV JAVA_HOME=/build/graal-jvmci-8/openjdk1.8.0_161/linux-amd64/product
 ENV JVMCI_VERSION_CHECK=ignore
 RUN $JAVA_HOME/bin/java -version
 
-# Build the Graal compiler
-RUN git clone https://github.com/oracle/graal.git
-RUN cd graal/compiler && mx build
-ENV GRAAL_HOME=/build/graal/compiler
-
 # Build TruffleRuby
 RUN git clone https://github.com/oracle/truffleruby.git
 RUN cd truffleruby && mx build
 
+# Build the Graal compiler
+RUN cd graal/compiler && mx build
+ENV GRAAL_HOME=/build/graal/compiler
```

Comment:
This will mean the `graal` version will be the `truffle` we import.
Probably better to ensure they work well together.
They lose the `--depth 1` but I don't see a way around that.